diff --git a/node_modules/@generousgames/mimi-pkg/dist/src/utils/exec.js b/node_modules/@generousgames/mimi-pkg/dist/src/utils/exec.js
index 3726454..5b0cd3b 100644
--- a/node_modules/@generousgames/mimi-pkg/dist/src/utils/exec.js
+++ b/node_modules/@generousgames/mimi-pkg/dist/src/utils/exec.js
@@ -20,7 +20,7 @@ async function run(cmd, args, opts) {
     return out.exitCode ?? 0;
 }
 async function ensureTool(name) {
-    const which = await (0, zx_1.$) `node -e "process.exit(!Boolean(require('node:child_process').spawnSync(process.platform==='win32'?'where':'which',['${name}'],{stdio:'ignore'}).status===0))"`;
+    const which = await (0, zx_1.$) `node -e "process.exit(require('node:child_process').spawnSync(process.platform==='win32'?'where':'which',['${name}'],{stdio:'ignore'}).status===0?0:1)"`;
     if (which.exitCode !== 0)
         throw new Error(`Missing required tool: ${name}`);
 }
